DigitalLearningSolutions.Web/Controllers/LoginController.cs

```diff
@@ -98,16 +123,28 @@ public IActionResult ChooseACentre(int centreId)
             {
                 sessionService.StartAdminSession(adminAccountForChosenCentre.Id);
             }
+
             return LogIn(adminAccountForChosenCentre, delegateAccountForChosenCentre, rememberMe);
         }
 
         private void SetTempDataForChooseACentre(bool rememberMe, AdminUser? adminUser,
-            List<DelegateUser> approvedDelegateUsers)
+            List<DelegateUser> approvedDelegateUsers, ChooseACentreViewModel chooseACentreViewModel)
         {
+            if (adminUser != null)
+            {
+                adminUser.ProfileImage = null;
+            }
+
+            foreach (var user in approvedDelegateUsers.Where(du => du.ProfileImage != null))
```

Comment:
Currently, no profile image is grabbed in GetDelegateUsersByUsername so approvedDelegateUsers will never have an entity where the profile image is not null. So this foreach loop doesn't really do anything, but acts as a preventative measure in case changes are made to GetDelegateUsersByUsername to grab the profile picture at some point in the future. Do we think we should keep it, or get rid of it and only implement it if it becomes necessary?

Reply:
I think we should actually create a new model, which only has the fields that we need to save in temp data. Then whichever fields we add to the admin and/or delegate db queries won't be affecting Choose a centre behaviour, because there's only a set of fields we map to what's saved in the tempdata.